src/app/pages/SearchResultsPage/useRedirectIfSingleResult.ts

```diff
@@ -33,7 +34,7 @@ export function useRedirectIfSingleResult(
         redirectTo = RouteUtils.getTransactionRoute(item, item.eth_hash || item.hash)
         break
       case 'account':
-        redirectTo = RouteUtils.getAccountRoute(item, item.address_eth ?? item.address)
+        redirectTo = RouteUtils.getAccountRoute(item, (item as RuntimeAccount).address_eth ?? item.address)
```

Comment:
Why is this cast necessary?

Reply:
Because the data can be either (consensus) `Account` or `RuntimeAccount`, and only `RuntimeAccount` has this field. This is not a problem, since we can use `address` as a fallback, but TS is complaining when accessing a field that might not be defined at all (in case of `Account`).